CSVConvert.py

```diff
@@ -393,24 +395,32 @@ def process_mapping(line, test=False):
         return value, elems
     return line, None
 
+
 def read_mapping_template(mapping_path):
     """Given a path to a mapping template file, read the lines and
     return them as an array."""
     template_lines = []
     try:
         with open(mapping_path, 'r') as f:
-            lines = f.readlines()
+            lines = csv.reader(f)
             for line in lines:
-                if line.startswith("#"):
+                if len(line) == 0:
                     continue
-                if re.match(r"^\s*$", line):
+                if line[0].startswith("#"):
```

Comment:
I'm not sure that it matters, but here you are comparing the first *character* of the string `line` with `"#"`, not the string `line`.

Reply:
ah, this is because the csv reader behaves a bit differently than the straight readlines. It reads each line of the csv and stores a list of values split by commas. So if the line doesn't have a comma, it will still be a list with one value.